eskipfile/watch_test.go

```diff
@@ -36,30 +36,36 @@ type watchTest struct {
 	routing *routing.Routing
 }
 
-func deleteFile() {
-	os.RemoveAll(testWatchFile)
+func deleteFile(t *testing.T) {
+	err := os.Remove(testWatchFile)
+	if err != nil {
+		t.Logf("Ignoring %v", err)
+	}
 }
 
-func createFileWith(content string) {
+func createFileWith(content string, t *testing.T) {
 	f, err := os.Create(testWatchFile)
 	if err != nil {
-		return
+		t.Fatal(err)
 	}
 
 	defer f.Close()
-	f.Write([]byte(content))
+	_, errw := f.WriteString(content)
+	if errw != nil {
```

Comment:
You can also do 2x and reuse the var name `err`:
```
if _, err := f.WriteString(content); err != nil {
```

Reply:
Changed to `writeErr`.

Reply:
I would prefer just `err`. If you use the mentioned line, then `err := ...` can be used more than once in a function.

Reply:
What is the point to reuse variable name (when we have more names available than there're atoms in universe)? I know that it is possible to redefine the name in if block but in this case it is only possible because first return value is ignored.
If you really _prefer_ them both to read the same - they are now ;)